fr/tri/Tri_par_insertion.md

```diff
@@ -0,0 +1,45 @@
+# Tri par insertion
+
+## Description
+
+### Principe
+
+Le principe du tri par insertion est de parcourir la liste de valeurs, en les classant par rapport aux valeurs déjà parcourues.  
+On trie la 2ème valeur par rapport à la 1ère, la 3ème par rapport à la 2ème et la 1ème, etc.
+
+### Complexités
+
+Lors de l'évaluation des complexités d'un algorithme de tri, il est nécessaire de prendre en compte non seulement le cas général, mais également le meilleur et le pire cas.
+
+* La complexité spatiale d'un tri par insertion est simple et est constante à travers les cas. Il s'agit de la liste des valeurs, car le tri est effectué directement dans la liste, plus une variable qui servira à retenir la valeur que l'on est en train de trier.
+* La complexité temporelle varie beaucoup selon les cas :
```

Comment:
Perhaps elaborate on:
* Which input constitutes the worst case / best case / average case (list of numbers sorted in reverse order / sorted in order / average permutation)?
* How is the complexity obtained? (having to shift all sorted elements when inserting in the worst case)

Reply:
I will, thanks